lib/process-manager.js

```diff
@@ -60,6 +66,68 @@ class ProcessManager {
         // Base Implementation
     }
 
+    /**
+     * General implementation of figuring out if the Ghost blog has started successfully.
```

Comment:
We can reconsider if we want the port polling implementation only in the systemd extension.
Local process manger overrides it currently. I am fine with both. My thinking is that other extensions will find that helpful for sure.